From: Mitch Williams <mitch.a.williams@intel.com>
Date: Sat, 10 May 2014 04:49:11 +0000
Subject: [PATCH v3 343/474] i40e/i40evf: remove chatty reset messages
Patch-mainline: v3.16-rc1
Git-commit: 63158f916f21b986743d0cb6f9e273d54a01ce7e (partial)
References: bsc#909484 FATE#317397

Both the PF side and the VF side of the VF reset process are too noisy.
We already warn the user that a reset is happening, and that is
sufficient.

Because some of these message are inside if statements, we have to
rejigger the brackets at the same time to keep our coding style
consistent.

Change-ID: Id175562fb0ec7c396d9de156b4890e136f52d5f4
Signed-off-by: Mitch Williams <mitch.a.williams@intel.com>
Signed-off-by: Jeff Kirsher <jeffrey.t.kirsher@intel.com>
Acked-by: David Chang <dchang@suse.com>
---
 drivers/net/ethernet/intel/i40e/i40e_main.c |    2 --
 1 file changed, 2 deletions(-)

--- a/drivers/net/ethernet/intel/i40e/i40e_main.c
+++ b/drivers/net/ethernet/intel/i40e/i40e_main.c
@@ -5618,8 +5618,6 @@ static void i40e_reset_and_rebuild(struc
 	/* tell the firmware that we're starting */
 	i40e_send_version(pf);
 
-	dev_info(&pf->pdev->dev, "reset complete\n");
-
 end_core_reset:
 	clear_bit(__I40E_RESET_RECOVERY_PENDING, &pf->state);
 }
